data/web/public/index.php

```diff
@@ -1,127 +1,273 @@
         }
-    </style>
-</head>
-<body>
-<header>
-    <nav class="navbar" role="navigation">
-        <div id="vagrant"><div class="text-center"><h3>test environment for <?php echo $ready_name; ?></h3></div></div>
-        <div class="container">
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="sr-only">Toggle navigation</span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a class="navbar-brand" href=""><img alt="Hypernode" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARoAAABgCAMAAAD1o02bAAAAM1BMVEUAAAAngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr0ngr1G2G0/AAAAEHRSTlMAv0CA7xAgn2DfzzCPcFCvNjJDRwAABLtJREFUeF7tm2mTsyoQRpt1UdT+/7/2TsjyBMGKiXMnbwnnw5QFTEdPIc2S0EvkpJQfaUXHOs2JoAyBjnUMdF1OFwM5HblwiY62ixG8gbNdTEmX4wW/wI1tigm8AyG7mESXYzbEdDlK85uEmVrADPwBrmUz3c3CHzLRyfH8KdrSuQlcp79Skj9H06lRXGHQvCIEriDpzAgu8UXWckSuuYFYc8FSvGfaEJlqyxMzckma6IZivI1coFtL3bZ40RT9MFWbnhfHJVSokVu5zNN5CcfURDotho+pGei0zAfVsKGzEvnYMMyyoQnfdvJ2XEHRWeEa4uWUD4jG1paebOAMZ4xra4U5cZ1QKtBcZ2xtg69v9Wmu07ezLB8n9G3hTexZJ3zHmemMDPwLRNpkFD+M6TIKIa4tL2ULJebLdVIrlUg3o8WEXhjFHcQZHw0VGqaYKl35yyUq/KWxdmg6O6GZOVw+B4hnItaWBxG0icRSQjxaOqT8dN9mvUPtTHWqLmndMJps5mof9cXho/ZUxlwgpz7jPc6basZHXrPodIoBD2afGhaZGgc1MHPHV3TrcVON4l9BvqcmXeqHD1uqYbFPDR6YE3alJjLQthJT239Mjb8P3gEb7xc1g5TSC851DvKOgRoiM6eGQ6bGoR5TEzdP+laJmHMMNwtQM8kbI5YJRxnfVJOUuNubJaFGQJxb6QR49AUSiK9Y1N8fT9FtYawpjzk933vR+Y3+82EYj2fIXSeMUIO0KXaomZOMTI271SNQgO4ZMWEu5mrAKPgwzrytxlyHCZ3+FmrcO2pMpobttR6FER+o1jEDLqHmD4Aapy7c7wNPP6fM/arXBHXF73mh2F3rkQlnxFzWamJu0akr9u/UAJGdCw7oz8VYE4tsIlZq7PJ8evrYSLFQgy4LIZmarIMB+U01eGi7VmMUFh/baoQI9wwANaljujfUzP+kGo8jc6jRQgxouK0GeHpWk4K6/Wqmr6sZxAUNNdiUl5ma6vRMY2FTNIyUqbkG/fiFGrBWqyDVa6biX9/MUFARqKpmSJ+wmaFAWKnxUIPChMb4hZgDQsDisW+CusNqkE5LNYMneqVGSimw+QoLIVMT8mXbhJi4uWWfGsc78cfVUKkm5WlpiV6ruYfWJo/mMzUuWXzkeYuY6Ah+nxrey/I/qEH1PjW0IAKiDVBztyfJxNQbs5jWa7j9RTXiD9UAkauxGLARTXICasFcizlVH11+vucXv60G74tbRRPPakYGSy2mo7qaA2PN19VgcWwRDTOp8lhgMJWYkfaq8bwT+3U1uFwQDTrX62etypjaWdqtZvz1kyeDLagx7RGBSw0Uk0U1GOUz46OhJESXiIZ6AtYrJVf3pNS0+jCZYahAH0xQ/ScuE7WHOrjh2X/JQS3yOxO+dg96FbVI5B3M1CL2yKymp29PbWJ0H4S3GF+4GQx1N91MgXG8hVbUODZwFWWoeQRXkdQRxS+9Q3io6WqUUYFvCE8CaroaIpqj+EGNRJmargZ0NV1NV9PVdDVdTVfT1XQ1XU0L4Dt9riyaqHWU5gthrhU1jSvPDkRW1CyqPIyLxXZWP6QLZdFAzTKVXxdRZVE/vuQJI00fbaJ4xteKmqbzH5wjP7BojybsAAAAAElFTkSuQmCC" /></a>
+
+    }
+    return $config;
+}
+
+/**
+ * Formats an URL which redirects to the PHP info page.
+ * @return string
+ */
+function getPHPInfoPage(){
+    $phpInfoPage = "https://";
+    if(!$_SERVER['HTTPS']){
+        $phpInfoPage = "http://";
+    }
+    $phpInfoPage .= $_SERVER['HTTP_HOST'].$_SERVER['PHP_SELF']."?info=true";
+    return $phpInfoPage;
+}
+
+if($_GET['info']):
```

Comment:
No it isn't. It's a developer practice to use this way instead of the accolades in files which also have HTML in it. It provides more readability and is common practice in Magento's .phtml files. It isn't required, just a matter of preference. i think it's more clean to do:

``` php 
<?php if($someStmt): ?>
<div class="myhtml">data</div>
<?php endif; ?>
```

instead of

``` php 
<?php if($someStmt){ ?>
<div class="myhtml">data</div>
<?php } ?>
```

because the endif; line is more obivous in code that the condition ends there when mixed with HTML.

Here's a little info about the alternative syntax.

[PHP Alternative Syntax](http://php.net/manual/en/control-structures.alternative-syntax.php)